Source/DafnyTestGeneration/TestMethod.cs

```diff
@@ -640,6 +696,9 @@ public class TestMethod {
       }
 
       lines.Add($"method {{:test}} test{id}() {{");
+
+      // TODO: The lines below for constructing the test input are duplicated above in TestInputConstructionLines
+      // Ideally we should move this code there permanently and call it from here.
```

Comment:
`TestInputConstructionLines` does not alter the logic in any way, so I think we should be able to do this.